### ISSUE TITLE
Plugins cannot be installed on fresh install of hombridge

### ISSUE THREAD

**Issue by WirthmU** [homebridge/homebridge#3199]
### Analysis

I did a complete fresh install of hombridge and wanted to add plugins - none of them is working.

### Expected Behavior

Plugins were installed before.

### Steps To Reproduce

homebridge has been re-installed

### Logs

```markdown
Setup Payload:
X-HM://00249LGQU7DBV
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 481-44-646 │     
    └────────────┘     
                       
[7/26/2022, 9:49:24 AM] Homebridge v1.5.0 (HAP v0.10.2) (Homebridge 331F) is running on port 51329.
[7/26/2022, 9:50:27 AM] [Homebridge UI] Added new user: admin
[7/26/2022, 9:51:20 AM] [Homebridge UI] Running Command: /opt/homebridge/lib/node_modules/homebridge-config-ui-x/plugin-upgrade-install.sh homebridge-fhem 0.5.36 /var/lib/homebridge/node_modules
[7/26/2022, 9:51:36 AM] [Homebridge UI] Running Command: npm install --save homebridge-fhem@0.5.36
[7/26/2022, 9:51:37 AM] [Homebridge UI] Operation failed. Please review log for details.
[7/26/2022, 9:54:36 AM] [Homebridge UI] Running Command: npm install --save homebridge-homematic@0.2.226
[7/26/2022, 9:54:37 AM] [Homebridge UI] Operation failed. Please review log for details.
[7/26/2022, 9:57:31 AM] [Homebridge UI] Running Command: npm install --save homebridge-withings-air-quality@0.1.4
[7/26/2022, 9:57:32 AM] [Homebridge UI] Operation failed. Please review log for details.
```
```
The log-file does not provide any more details about why the installation failed!

### Configuration

```json
{
    "bridge": {
        "name": "Homebridge 331F",
        "username": "0E:32:F7:ED:33:1F",
        "port": 51329,
        "pin": "481-44-646",
        "advertiser": "bonjour-hap"
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        }
    ]
}
```

### Environment

- OS: Ubuntu 20.04.4 LTS
- Software: v16.16.0 
- Node: v18.6.0
- npm: 8.13.2

### Process Supervisor

hb-service

**Comment by WirthmU** [comment]
Plugins could be loaded using CLI but not from the homebridge web interface:
sudo npm install --save homebridge-dyson-pure-cool@2.7.3

**Comment by AMoo-Miki** [comment]
I faced the same problem on a fresh install of `homebridge` on a pi using `apt`. Found that, when called from the UI terminal, `npm` exited prematurely without doing anything; even `npm -v` didn't produce an output.

`npm@8.11.0` which was installed by `homebridge` was trying to `mkdir '/home/homebridge/.npm'` when `/home/homebridge` was owned by `root:root` and this was causing an `EACCES` which was not finding its way to the output.

Two things worked for me:
* `sudo chown -R homebridge:homebridge /home/homebridge` via the normal terminal, or
* bumping to 8.17.0
